### PR TITLE
Changing default enpoint from platform to analytics [COM-1098]

### PR DESCRIPTION
**Pull request by toussaints**
To prevent the platform as a whole to be blocked, they have separated the endpoint for the analytics. Doc have been updated and the endpoints are all live now.

### CODE REVIEW

**Review by francoislg — APPROVED** [review]
Change looks good

File: src/client/analytics.ts [hunk 1]
```diff
@@ -200,47 +200,41 @@ export class CoveoAnalyticsClient implements AnalyticsClient, VisitorIdProvider
...
-    async resolveParameters(eventType: EventType | string,
-        ...payload: VariableArgumentsPayload) {
-            const {
-                variableLengthArgumentsNames = [],
-                addVisitorIdParameter = false,
-                usesMeasurementProtocol = false,
-            } = this.eventTypeMapping[eventType] || {};
+    async resolveParameters(eventType: EventType | string, ...payload: VariableArgumentsPayload) {
+        const {variableLengthArgumentsNames = [], addVisitorIdParameter = false, usesMeasurementProtocol = false} =
+            this.eventTypeMapping[eventType] || {};
```
[the remark this answers is not in the document]
Comment (toussaints) [reply src/client/analytics.ts:205]: No, it was vscode that did it, but now that I did run lint fix there is even more changes 😨

[the remark this answers is not in the document]
Comment (toussaints) [reply src/client/analytics.ts:205]: See https://github.com/coveo/coveo.analytics.js/pull/282/commits/c10a6ce5add75240292083e84ede4dde04d38f48

**Review by cnguyen-coveo — APPROVED** [review]

File: src/client/analytics.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (francoislg) [reply src/client/analytics.ts:205]: Ok then, no worries

**Pull request merged**